packages/alert/src/index.js

```diff
@@ -0,0 +1,83 @@
+//@flow
+import type { StatelessFunctionalComponent, Node } from "react";
+import type { AlertType } from "./AlertTypes";
+import React from "react";
+import styled from "styled-components";
+import AlertTypes from "./AlertTypes";
+
+type AlertComponent = StatelessFunctionalComponent<{
```

Comment:
Let's simplify this component...
Take a look at this [stateless component](https://flow.org/en/docs/react/components/#toc-stateless-functional-components) and this [default props](https://flow.org/en/docs/react/components/#toc-using-default-props-for-functional-components). It's so much easier to read (: